src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -139,6 +139,10 @@ public class DrivebaseSubsystem extends SubsystemBase {
 	public DrivebaseSubsystem() {
 		gyroscope = isComp ? new Pigeon2Gyro(Hardware.GYRO_PORT) : new NavXGyro(SerialPort.Port.kMXP);
 
+		if (isComp) {
+			gyroscope.setInverted(true);
+		}
```

Comment:
This seems strange, as the gyro is currently mounted right-side up. Please add a comment for why this is needed